### PR TITLE
Static constants feature

### PR DESCRIPTION
**Pull request by fbudassi**
I have been working with your library for a couple of months, but after trying to use $statics to add immutable constants and object literals grouping immutable constants to a project, I realized you could accidentally modify them.

I have  made a simple implementation that requires ES5 to work. I hope you consider it interesting enough to be merged to the master branch of your library. It includes a simple sample and the corresponding unit tests together with the main implementation.

Don't hesitate to contact me if you think the code can be improved.

Regards,
Federico Budassi.

### CODE REVIEW

File: jsface.js [hunk 1]
```diff
@@ -174,6 +195,15 @@
       clazz[key] = statics[key];
     }
 
+    // copy immutable properties from consts to clazz and freeze them recursively
```
Comment (tnhu) [inline jsface.js]: Extract this logic to $const plugin under Class.plugins. See: https://github.com/tnhu/jsface/blob/master/jsface.js#L190 as a reference.

**Comment by tnhu** [comment]
Hi Federico,

$const looks great. Can you refactor the code to make $const is a plugin instead (I made a comment on your change). I'll merge the code and update README.

Thanks,
Tan

**Comment by fbudassi** [comment]
Hi Tan,

Done, $const moved to the Class.plugins.

Regards,
Federico.

**Comment by tnhu** [comment]
Merged. This is a nice feature :+1: Thanks Federico!

Tan

**Comment by fbudassi** [comment]
You're welcome.

Federico.

**Pull request merged**